docs/zh/guide/deployment.md

```diff
@@ -82,10 +82,8 @@ serve -s dist
 
     cd -
     ```
-
-    ::: tip
-    You can also run the above script in your CI setup to enable automatic deployment on each push.
-    :::
+    
+    你还可以选择在你的CI设置中运行上述脚本，便于在每次推送时启用自动部署。
```

Comment:
```
::: tip
:::
```
This indicates a tip section in `vuepress`, which should not not be omited.

Reply:
ok, thank you